src/main/java/com/endava/Application.java

```diff
@@ -1,19 +1,53 @@
 package com.endava;
 
 import java.io.IOException;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
 
 public class Application {
 
+    private static final String PATH1 = "./src/main/resources/menu1.csv";
+    private static final String PATH2 = "./src/main/resources/menu2.csv";
+
     public static void main(String[] args) throws IOException {
         double start = System.currentTimeMillis();
 
-        double menu1AveragePrice = FileUtil.getPriceAverage("./src/main/resources/menu1.csv");
-        double menu2AveragePrice = FileUtil.getPriceAverage("./src/main/resources/menu2.csv");
+        double menu1AveragePrice = FileUtil.getPriceAverage(PATH1);
+        double menu2AveragePrice = FileUtil.getPriceAverage(PATH2);
 
         double end = System.currentTimeMillis();
 
         System.out.println("Menu 1 average price: " + menu1AveragePrice);
         System.out.println("Menu 2 average price: " + menu2AveragePrice);
         System.out.println("Time to process: " + (end - start) / 1000 + " seconds");
+
+        start = System.currentTimeMillis();
+
+        ExecutorService executorService = Executors.newFixedThreadPool(2);
+
+        Runnable runnableTask = () -> {
+
+            double averagePrice1 = 0;
+            double averagePrice2 = 0;
+
+            try {
+                averagePrice1 = FileUtil.getPriceAverage(PATH1);
+                averagePrice2 = FileUtil.getPriceAverage(PATH2);
+            } catch (IOException e) {
+                e.printStackTrace();
+            }
+
+            System.out.println("Menu 1 average price: " + averagePrice1);
+            System.out.println("Menu 2 average price: " + averagePrice2);
+        };
+
+        executorService.execute(runnableTask);
```

Comment:
You should have 2 Runnables and execute them with ExecutorService.
The current implementation has only one task and it's executing it in one thread. (1 Runnable = 1 task)